source/computer/wsl-setup.rst

```diff
@@ -23,9 +23,14 @@ WSL 有 WSL1 和 WSL2 两个发行版本，二者底层原理不同。大多数
 
 .. warning::
 
-   同时使用 WSL 和 VMware/VirtualBox 的用户建议只启用 WSL1。
-   经测试，开启 WSL2 后 VMware/VirtualBox 无法正常使用，
-   关闭 WSL2 功能后 VMware/VirtualBox 恢复正常。
+   由于 Hyper-V 兼容性问题，开启 WSL2 功能后 VMware/VirtualBox 将无法正常使用，
+   而 WSL1 和 VMware/VirtualBox 不存在兼容性问题，可同时运行。
+   
+   已开启 WSL2 功能的用户若需要使用 VMware/VirtualBox，可以先把 Linux 发行版改为 WSL1，
+   然后使用管理员模式打开 PowerShell，并输入 ``bcdedit /set hypervisorlaunchtype off`` 。
+   重启后生效，此时 Hyper-V 功能关闭，VMware/VirtualBox 可用，WSL2 不可用。
+   用户同样可以在 PowerShell 中输入 ``bcdedit /set hypervisorlaunchtype auto`` ，
+   重启后生效，此时 Hyper-V 功能开启，WSL2 可用，VMware/VirtualBox 不可用。
```

Comment:
After we restart Hyper-V, don't we go back to WSL1 since we first change WSL2 to WSL1?
```suggestion
   用户同样可以在 PowerShell 中输入 ``bcdedit /set hypervisorlaunchtype auto``
   开启 Hyper-V，重启后方能生效。此时 Hyper-V 功能开启，WSL2 可用，
   VMware/VirtualBox 不可用。
```

Reply:
You can run `wsl --set-version` command to change 1 to 2 after restart Hyper-V.

Reply:


> You can run `wsl --set-version` command to change 1 to 2 after restart Hyper-V.

Yes, I just make a suggestion. You may choose to add it in the manual.  I don't have a Windows~